site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -207,6 +207,22 @@
         dropzone.addEventListener("dragleave", draghandle, false);
         dropzone.addEventListener("drop", dropWithMultipleZips, false);
     </script>
+    <script>
```

Comment:
Why are there three different sets of `<script>` tags back to back here?  All of this should be combined into one single `<script>` tag.

Ideally some of this JS code should be moved to a `.js` file if possible (everything except twig-rendered data), but splitting that into a separate file isn't necessarily related to this change, and should be done in a separate PR.